Makefile

```diff
@@ -205,7 +202,7 @@ verify: .init .generate_files verify-generated verify-client-gen verify-vendor
 	   golint --set_exit_status $$i || exit 1; \
 	  done'
 	@#
-	$(DOCKER_CMD) go vet $(NON_VENDOR_DIRS)
+	$(DOCKER_CMD) go vet ./...
```

Comment:
Or, would we rather see `$(SC_PKG)/...` ?

Reply:
Looks like everywhere else in the Makefile we are using `$(SC_PKG)` for the docker commands.

Personally I love to  `./...` all the things, but I assume that we are doing this to avoid questions around the current working directory in the docker container.

Reply:
Ya, I'll see if this builds once in travis and then switch it over to `$(SC_PKG)`/

Reply:
success. replacing